data/FilterList.json

```diff
@@ -9602,7 +9602,9 @@
     "licenseId": 28,
     "name": "abuse.ch URLhaus Malicious URL Blocklist",
     "syntaxId": 2,
-    "viewUrl": "https://gitlab.com/curben/urlhaus/raw/master/urlhaus-filter.txt"
+    "viewUrl": "https://gitlab.com/curben/urlhaus-filter/raw/master/urlhaus-filter.txt",
+    "viewUrlMirror1": "https://glcdn.githack.com/curben/urlhaus-filter/raw/master/urlhaus-filter.txt",
+    "viewUrlMirror1": "https://cdn.staticaly.com/gl/curben/urlhaus-filter/raw/master/urlhaus-filter.txt"
```

Comment:
Please change the second mirror to `viewUrlMirror2` (notice the trailing "2"). Thanks.

Reply:
ah, missed that. fixed.